src/main/java/frc/robot/vision/sources/VisionSource.java

```diff
@@ -0,0 +1,13 @@
+package frc.robot.vision.sources;
+
+import frc.robot.vision.RawVisionAprilTagData;
+
+import java.util.Optional;
+
+public interface VisionSource<T> {
```

Comment:
why do you have T if you don't use it?

Reply:
The getRawVisionEstimation() method should return an optional of T, I will fix this at some point